src/components/Image/Image.docs.mdx

```diff
@@ -0,0 +1,42 @@
+import { Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import Image from './Image';
+
+# Image
+
+Displays an image. The `Image` component requires `height` and `width` to set aspect ratio and `sizes` and `srcSet` to provide [responsive rendering](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images). They will not be calculated for you and there are no default values. Future components may provide a simpler implementation for specific use cases.
+
+This component does not provide [art direction](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture), which is also a task for a future component.
+
+## Usage
+
+```js
+import { Image } from '@quartz/interface/src/components';
+```
+
+## Props
+
+<Props of={Image} />
+
+## Usage guidelines
+
+- **Do** use a container to constrain the size or shape of the image. An `Image` will always expand or contract to fit the width of its container. The `Image` component purposefully does not accept a `className` prop.
+- **Do** understand that the `height` and `width` props are [used to help the browser reserve space](https://blog.logrocket.com/jank-free-page-loading-with-media-aspect-ratios/) for the image. They do not need to match the _rendered_ size of the image but should be set at reasonable upper bounds.
+- **Do** [provide useful alt text](https://www.w3.org/WAI/tutorials/images/decision-tree/) for your images. Sometimes, an empty string is the most useful!
+- **Do** learn why and how to use [responsive rendering](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images). It’s important to understand how `sizes` and `srcSet` work.
+- **Do** think about AMP when using the `Image` component and pass the correct `amp` prop.
```

Comment:
Nitpick but could you elaborate on 'think about AMP'? Something like 'think about whether the image will appear on a page rendered by Google AMP e.g. an article page'